Resources/Prototypes/_SpaceCats/Catalog/VendingMachines/daki.yml

```
@@ -0,0 +1,439 @@
+      stock: 5
+
+- type: listing
+  id: Daki40
+  productEntity: Pillow6
+  cost:
+    SpaceCash: 100
+  categories:
+    - Daki
+  conditions:
+    - !type:ListingLimitedStockCondition
+      stock: 5
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)